src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -197,7 +197,8 @@ class QueueIO[T <: Data](private val gen: T, val entries: Int) extends Bundle
 class Queue[T <: Data](val gen: T,
                        val entries: Int,
                        val pipe: Boolean = false,
-                       val flow: Boolean = false)
+                       val flow: Boolean = false,
+                       val useSyncReadMem: Boolean = false)
```

Comment:
Should update the scaladoc above

Reply:
Sorry, completely forgot about that one. It's now updated.